packages/web/src/repositories/position/position.message.ts

```diff
@@ -485,8 +482,8 @@ export function makeRepositionLiquidityMessagesWithApproves(
       `${maxTick}`, // position's maximal tick
       `${tokenAAmountRaw}`, // Maximum amount of tokenA to offer
       `${tokenBAmountRaw}`, // Maximum amount of tokenB to offer
-      BigNumber(tokenAAmountRaw).multipliedBy(slippageRatio).toFixed(0), // Minimum amount of tokenA to provide
-      BigNumber(tokenBAmountRaw).multipliedBy(slippageRatio).toFixed(0), // Minimum amount of tokenB to provide
+      "0", // Minimum amount of tokenA to provide
+      "0", // Minimum amount of tokenB to provide
```

Comment:
setting the minimum token provision to 0 would likely risk users receiving far fewer tokens than expected, or completely disable the slippage protection.